File: content/54/a.php
<?php
$s = 'https://speedtest.lax.hivelocity.net';
# example 1
$s2 = file_get_contents($s);
# example 2
$r = fopen($s, 'r');
$s3 = stream_get_contents($r);
# print
var_dump($s2, $s3);

File: content/54/a.dart
import 'dart:io';

void main() {
   var o_uri = Uri.parse("https://speedtest.lax.hivelocity.net");
   var o_http = new HttpClient();
   o_http.getUrl(o_uri).then(
      (o_req) => o_req.close()
   ).then(
      (o_resp) => o_resp.pipe(stdout)
   );
   o_http.close();
}
